### PR TITLE
Document table diffs

### PR DESCRIPTION
**Pull request by robertkowalski**
This adds the differences I found for `console.table`.

Parent issue: #21

Sunny greetings from the Canary Islands! :palm_tree: :sunny:

### CODE REVIEW

**Comment by terinjokes** [comment]
Good idea moving those table notes over here. (Y)

File: NOTES.md [hunk 1]
```diff
@@ -107,3 +107,78 @@ Chrome:
...
 ```
+
+## console.table - printing of strings
+
+```
+FF:
+
+console.table("the plaice living on the balcony")
+undefined
+console.table():
```
Comment (terinjokes) [inline NOTES.md]: this a mistake?

Comment (robertkowalski) [inline NOTES.md]: No that is the output... I can will add the screenshot for clarification

screen shot 2016-02-21 at 21 19 34

**Comment by robertkowalski** [comment]
@terinjokes added screenshots for the string-case

File: NOTES.md (hunk 1 above)
Comment (terinjokes) [inline NOTES.md]: That's, uh, surprising.

**Pull request merged**